lib/ui/views/app_selector/app_selector_viewmodel.dart

```diff
@@ -93,11 +95,13 @@ class AppSelectorViewModel extends BaseViewModel {
     final app =
         await DeviceApps.getApp(packageName, true) as ApplicationWithIcon?;
     if (app != null) {
-      if (await checkSplitApk(packageName) && !isRooted) {
-        return showSelectFromStorageDialog(context);
-      } else if (!await checkSplitApk(packageName) || isRooted) {
+      if (!await checkSplitApk(packageName) ||
+          isRooted ||
+          _managerAPI.getBypassSelectFromStorage()) {
         selectApp(app);
         Navigator.pop(context);
+      } else if (await checkSplitApk(packageName) && !isRooted) {
```

Comment:
This would work as just an else clause

Reply:
Resolved in 114f2506d5b0868749b54cc44d0dee550d633d41